boot/initramfs20dirs.go

```diff
@@ -82,6 +86,7 @@ func setInitramfsDirVars(rootdir string) {
 	InitramfsUbuntuSaveDir = filepath.Join(InitramfsRunMntDir, "ubuntu-save")
 	InstallHostWritableDir = filepath.Join(InitramfsRunMntDir, "ubuntu-data", "system-data")
 	InstallHostFDEDataDir = dirs.SnapFDEDirUnder(InstallHostWritableDir)
+	InstallHostFDESaveDir = filepath.Join(InitramfsUbuntuSaveDir, "device/fde")
```

Comment:
this looks right - now, but is always correct essentially because we assume that if encrypted we have a ubuntu-save partition. As @cmatsuoka mentioned we probably need to add a check about that earlier in install though, that if encrypted we should fail if there's no ubuntu-save partition